### PR TITLE
Add PNG output to test_ReadPixels.

### PR DESCRIPTION
**Pull request by prideout**

### CODE REVIEW

File: filament/backend/test/test_ReadPixels.cpp [hunk 1]
```diff
@@ -16,15 +16,38 @@
...
 #include <fstream>
 
+using namespace filament;
+using namespace filament::backend;
+using namespace image;
+
 namespace {
 
+template<typename T>
+static LinearImage toLinear(size_t w, size_t h, size_t bpr, const uint8_t* src) {
```
Comment (romainguy) [inline filament/backend/test/test_ReadPixels.cpp:37]: Don't we have multiple copies of this method over the code base now? Worth having only one?

**Review by romainguy — APPROVED** [review]

File: filament/backend/test/test_ReadPixels.cpp (hunk 1 above)
Comment (prideout) [reply filament/backend/test/test_ReadPixels.cpp]: Good catch, added a commit.

**Comment by prideout** [comment]
Hm, iOS is failing, I'll need to figure that out.  This PR isn't really important, I found it somewhat useful though.

File: filament/backend/CMakeLists.txt [hunk 2]
```diff
@@ -307,6 +307,8 @@ if (APPLE)
         filabridge
         getopt
         gtest
+        image
+        imageio
```
Comment (bejado) [inline filament/backend/CMakeLists.txt:311]: You should also add these to `BACKEND_TEST_DEPS`.

**Review by bejado — CHANGES_REQUESTED** [review]
The iOS build is failing because the `clang` invocation is missing the `imageio` include directory. Not sure why though..

**Pull request merged**